examples/ingress_canary_and_auth.ipynb

```diff
@@ -0,0 +1,1088 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c4622ff6-be0d-4e8a-95c7-24c3140b488f",
+   "metadata": {},
+   "source": [
+    "# NOTE:\n",
+    "\n",
+    "\n",
+    "This guide requires the seldon-core charm revision >55 and istio-pilot charm revision >62.  As these are rolled out, you may need to deploy from the edge channel (`--channel latest/edge`) or build locally (`charmcraft pack`) for those charms.  See [seldon-core](https://charmhub.io/seldon-core) and [istio-pilot](https://charmhub.io/istio-pilot) in Charmhub to check the available versions."
```

Comment:
I wouldn't add the revision number here, but a supported channel instead.

Reply:
I agree in principle, but the revision we depend on here is newer than edge (the versions we need are still in branches).  I was relying on some luck though, as >55/>62 might not be enough if we push something else to one of those charms without fixing the things this demo needs :)  I'd love suggestions on a better way to phrase it, I just couldn't think of any

Reply:
This is even worse than that, if we push a new revision for istio 1.5 the revision would be >62 but an entirely different charm. We can never rely on a range of revision numbers. So we either hard pin it to 55 and 62 or make the effort of pushing it to a channel as soon as possible.

Reply:
Very true.  What do you propose as improved language?  It seems wrong to have no warning at all, but until istio is patched we can't actually say what version is needed.  Maybe something like "requires istio-pilot from a latest track that has a charm revision >62"?  More confusing, but more explicit